tests/test_primitive_play.py

```diff
@@ -17,30 +16,42 @@ def test_trick():
     return trick
 
 
-def test_create_deck():
+def test_create_deck_returns_randomized_deck_by_default():
     deck = create_deck()
+
+    # Hard to test for randomness...
     assert len(deck) == 24
 
 
-def test_create_players():
-    # tried to do this with deck as a fixture, but, when passed to create_players, it was treated as a function, not a list
-    card_values_and_ranks = {
-        "9": 9,
-        "10": 10,
-        "jack": 11,
-        "queen": 12,
-        "king": 13,
-        "ace": 14,
-    }
-    suits = ["clubs", "diamonds", "spades", "hearts"]
-    deck = []
-    for suit in suits:
-        for rank, value in card_values_and_ranks.items():
-            card = {"suit": suit, "value": value, "rank": rank}
-            deck.append(card)
-    shuffle(deck)
-
-    players = create_players(deck)
+def test_create_deck_supports_alternate_shufflers():
+    def null_shuffle(target):
+        # Could also seed the random number generator so you get deck shuffled,
+        # but the same way every time.
+        # import random
+        # random.seed(42)  # (value doesn't matter)
+        # create_deck(shuffle=random.shuffle)
+        pass
+
+    deck = create_deck(null_shuffle)
+
+    # Just check the first suit:
+    assert deck[:6] == [
```

Comment:
I might opt for this, because it's less fragile (will break for fewer reasons unrelated to the fundamental thing it's testing):
```python
    ranks_of_first_six_cards = [card["rank"] for card in deck[:6]]
    assert ranks_of_first_six_cards == [
        "9",
        "10",
        "jack",
        "queen",
        "king",
        "ace",
    ]
```